packages/cli/src/commands/generate-graphql.ts

```diff
@@ -1,26 +1,63 @@
 import { Command, Flags } from '@oclif/core'
-import { spawn } from 'child_process'
 import { existsSync } from 'fs-extra'
+import chalk from 'chalk'
+
 import { tmpDir } from '../utils/directory'
+import { runCommandSync } from '../utils/runCommandSync'
 
 export default class GenerateGraphql extends Command {
   static flags = {
     debug: Flags.boolean({ char: 'd' }),
+    core: Flags.boolean({ char: 'c', hidden: true }),
   }
 
   async run() {
-    if (!existsSync(tmpDir)) {
-      throw Error(
-        'The ".faststore" directory could not be found. When running faststore dev or faststore build, the generate-graphql command is automatically executed.'
+    const { flags } = await this.parse(GenerateGraphql)
+
+    const debug = flags.debug ?? false
+    const isCore = flags.core ?? false
+
+    if (!isCore && !existsSync(tmpDir)) {
```

Comment:
Why not create the `.faststore` directory at this point instead of error? I keep thinking about the "pit of success" and how giving an error here is just extra work for our user.

Reply:
I think my thought behind it was to avoid having to copy it all again when running `dev` or `build`, since this will run again. If this condition is true, this means the user doesn't have any previous `.faststore` in their project, which probably means they haven't executed it in any way yet. Doing this will absolutely nothing that wouldn't be done by running `dev` or `build`.

Despite all that, I agree with you. The user would perceive it better if it just generated it anyways. I'll fix this